docs/sources/flow/_index.md

```diff
@@ -59,11 +59,11 @@ prometheus.remote_write "default" {
 
 ## Next steps
 
-* [Install][] Grafana Agent Flow.
-* Learn about the core [Concepts][] of Grafana Agent Flow.
+* [Install][] Grafana Agent in flow mode.
+* Learn about the core [Concepts][] of Grafana Agent.
 * Follow our Grafana Agent Flow [Getting started][] guides.
-* Follow our [Tutorials][] to get started with Grafana Agent Flow.
-* Learn how to use Grafana Agent Flow's [Configuration language][].
+* Follow our [Tutorials][] to get started with Grafana Agent.
+* Learn how to use Grafana Agent's flow mode [Configuration language][].
 * Check out our [Reference][] documentation to find specific information you
   might be looking for.
```

Comment:
All of these are next steps for Flow mode, should we change them here to clarify?

for example, should "Learn about the core Concepts of Grafana Agent" say "Learn about the core Concepts of flow mode?"

Reply:
Yes, there's a lot of room for improving the link text. I've updated this part a little.